src/main/java/com/example/TicketController.java

```diff
@@ -0,0 +1,20 @@
+package com.example;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Optional;
+
+@RestController
+public class TicketController {
+
+    @Autowired
+    private TicketServiceRepo ticketService;
+
+    @GetMapping("/ticket")
+    public Optional<Ticket> getOneTicket() {
+        return Optional.ofNullable(ticketService.fetchTicketById(39));
+    }
```

Comment:
in the controller you should not use a repo, but a service. And use repo in this service.
You should also return prepared data for the controller (not Optional, but Ticket) from the service.
All this logic related to conversion from Optional to Ticket should be processed in the service